tv_apps/smart-system/js/action_menu.js

```diff
@@ -209,7 +218,7 @@
      * @return {Boolean} if the container is invisible, return false
      * @return {Number} if the container is visible, return z-index
      */
-    isVisible: function() {
+    isFocusable: function() {
       return this.container && this.container.classList.contains('visible');
```

Comment:
Seems `this.visible` can be reused here: `return this.container && this.visible;`